docs/dev/service_levels.md

```diff
@@ -134,3 +135,20 @@ Otherwise, e.g. if a role has multiple workload types declared,
 the conflicts are resolved as follows:
  - `X` vs `unspecified` -> `X`
  - `batch` vs `interactive` -> `batch` - under the assumption that `batch` is safer, because it would not trigger load shedding as eagerly as `interactive`
+
+### Effective service level
+
+User may have assigned multiple service levels by granting one role to another.
+
+As mentioned previously, any user may have assgined multiple service levels by granting one roles to another. To facilitate insight into which values come from which service level, there is `LIST EFFECTIVE SERVICE LEVEL OF <role_name>` command.
+
+The command displays a table with: option name, effective service level the value comes from and the option value.
+
+```
+> LIST EFFECTIVE SERVICE LEVEL OF username;
+
+ service_level_option | effective_service_level | value
+----------------------+-------------------------+-------------
+        workload_type |                     sl1 | unspecified
+              timeout |                     sl2 |          2s
+```
```

Comment:
As I asked before, I am wondering what was the inspiration for the command name "LIST EFFECTIVE SERVICE LEVEL OF". Is it based on some other database or similar to something else in Scylla?
I wonder (but don't really know) if it would make sense to use the "DESCRIBE" keyword for that.

Reply:
I've answered in comment above